pybloqs/block/base.py

```diff
@@ -460,7 +461,7 @@ def data(self):
         output = BytesIO()
 
         for child in container.children:
-            output.write(render(child))
+            output.write(render(child).encode())
```

Comment:
I think this might need to be:
```suggestion
            output.write(render(child).encode('utf-8'))
```

as `render` returns a unicode string:
https://github.com/manahl/PyBloqs/blob/8793cbf80b4a9deedcc1b9266c3fba91ba37e0ac/pybloqs/html.py#L38-L46